src/Admin/AdminInterface.php

```diff
@@ -725,6 +720,23 @@ public function showMosaicButton(bool $isShown): void;
      * Checks if a filter type is set to a default value.
      */
     public function isDefaultFilter(string $name): bool;
+
+    public function configureActionButtons(array $buttonList, string $action, ?object $object = null): array;
+
+    /**
+     * Check object existence and access, without throw Exception.
```

Comment:
Let's fix the grammar though
```suggestion
     * Check object existence and access, without throwing Exception.
```